qrisk/tests/test_stats.py

```diff
@@ -79,49 +137,140 @@ def test_annual_volatility(self, returns, period, expected):
+                   index=['one', 'two'])),
+        (df_week, 0.0, stats.WEEKLY,
+         pd.Series([0.093902996054410062, 0.037928477556776516],
+                   index=['one', 'two'])),
+        (df_month, 0.0, stats.MONTHLY,
+         pd.Series([0.045109540184877193, 0.018220251263412916],
+                   index=['one', 'two']))
     ])
-    def test_calmar(self, returns, expected):
-        self.assertEqual(
-            stats.calmar_ratio(
-                returns),
-            expected)
+    def test_downside_risk(self, returns, required_return, period, expected):
+        self.assertTrue(
+            np.all(
+                stats.downside_risk(
+                    returns,
+                    required_return=required_return,
+                    period=period) == expected))
 
     @parameterized.expand([
-        (pd.Series(px_list,
-                   index=dt), 0.0, 2.0)
+        (-simple_rets[:5], 0.0, stats.DAILY, -12.29634091915152),
+        (-simple_rets, 0.0, stats.DAILY, -1.2296340919151518),
+        (simple_rets, 0.0, stats.DAILY, np.inf),
+        (df_simple, 0.0, stats.DAILY,
+         pd.Series([3.0639640966566306, 38.090963117002495],
+                   index=['one', 'two'])),
+        (df_week, 0.0, stats.WEEKLY,
+         pd.Series([0.63224655962755871, 7.8600400082703556],
+                   index=['one', 'two'])),
+        (df_month, 0.0, stats.MONTHLY,
+         pd.Series([0.14590305222174432, 1.8138553865239282],
+                   index=['one', 'two']))
+
     ])
-    def test_omega(self, returns, annual_return_threshhold, expected):
-        self.assertEqual(
-            stats.omega_ratio(
-                returns,
-                annual_return_threshhold=annual_return_threshhold),
-            expected)
+    def test_sortino(self, returns, required_return, period, expected):
+        self.assertTrue(
+            np.all(
+                stats.sortino_ratio(
+                    returns,
+                    required_return=required_return,
+                    period=period) == expected))
 
     @parameterized.expand([
-        (-simple_rets[:5], -12.29634091915152),
-        (-simple_rets, -1.2296340919151518),
-        (simple_rets, np.inf)
+        (simple_rets, simple_benchmark, 0.076577237215895003)
     ])
-    def test_sortino(self, returns, expected):
+    def test_information_ratio(self, returns, factor_returns, expected):
         self.assertAlmostEqual(
-            stats.sortino_ratio(returns),
-            expected, DECIMAL_PLACES)
+            stats.information_ratio(returns, factor_returns),
+            expected,
+            DECIMAL_PLACES)
 
-    def test_tail_ratio(self):
-        returns = np.random.randn(10000)
+    @parameterized.expand([
+        (simple_rets, simple_benchmark, (0.00020161290322580659,
+                                         3.3266129032258061))
+    ])
+    def test_alpha_beta(self, returns, benchmark, expected):
```

Comment:
maybe we should test `alpha` and `beta` with `alpha_beta` to ensure that the results are the same.